assets/Js/analog_clock1.js

```diff
@@ -12,4 +12,21 @@
                     hr.style.transform = `rotateZ(${(hh)+(mm/12)}deg)`;
                     mn.style.transform = `rotateZ(${mm}deg)`;
                     sc.style.transform = `rotateZ(${ss}deg)`;
-                });
\ No newline at end of file
+                });
+
+const checkbox = document.getElementById("checkbox");
+const body = document.querySelector("body");
+const label = document.querySelector(".label");
+const root = document.querySelector(":root");
+checkbox.addEventListener("change", function() {
+    if (this.checked) {
+        body.style.background = "black";
+        label.style.backgroundColor = "#40DCA5";
+        root.style.setProperty('--sec-color', '#fff');
+    } else {
+        body.style.background = "skyblue";
+        label.style.backgroundColor = "white";
+        root.style.setProperty('--sec-color', 'red');
+
+    }
+});
```

Comment:
```suggestion
const checkbox = document.getElementById("checkbox");
const body = document.querySelector("body");
const label = document.querySelector(".label");
const root = document.querySelector(":root");
const ball = document.querySelector(".ball");

checkbox.addEventListener("change", function () {
  if (this.checked) {
    body.style.background = "black";
    label.style.backgroundColor = "#40DCA5";
    root.style.setProperty("--sec-color", "#fff");
    ball.style.backgroundColor = "#fff"; // Change ball color to white
  } else {
    body.style.background = "skyblue";
    label.style.backgroundColor = "white";
    root.style.setProperty("--sec-color", "red");
    ball.style.backgroundColor = "black"; // Change ball color to black
  }
});
```

This will change the color on toggle of checkbox, which is good for accessibility and contrast